README.md

```diff
@@ -734,6 +735,24 @@ The best practice to determine top candidates for changing materialization from
 - if a view is used downstream my *many* models, change materialization to table
 - if a view has more complex calculations (window functions, joins between *many* tables, etc.), change materialization to table
 
+### Exposure Materializations
```

Comment:
Do you need to add a link to this section in the table of contents?

Reply:
Also do we want to name this Exposure Parents Materializations to match the model name?

Reply:
we should do both!